src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -296,9 +331,16 @@ else if (lineNo > childAstLineNoMap.get(MAX)) {
         }
     }
 
-    private boolean isMatchingAst(DetailAST root, TreeWalkerAuditEvent event) {
-        return root.getType() == event.getTokenType()
-                && root.getLineNo() == event.getLine()
-                && root.getColumnNo() == event.getColumnCharIndex();
+    /**
+     * Check whether ast node is matching with event's node.
```

Comment:
`Check whether ast node is matching with event's node.`
-->
`Check whether AST node matches event's node.`